### PR TITLE
Increased width of activity boxes

### LINKED ISSUE #168: Increase the width of the activity boxes to align it with the one above
The appearance of the activity boxes which are displayed when the user clicks on a particular day can be modified. 
Screenshot from 2020-03-24 23-41-59

1) The heading 'XYZ's activity on date' is in very small font which can be changed.
2) The boxes aren't aligned/centered appropriately. The width of the boxes can be increased.

### PR DESCRIPTION
**Pull request by eeshadutta**
Fixed #168

### CODE REVIEW

**Comment by rammanoj** [comment]
@eeshadutta I see a lot of change in formatting. Can you please remove it and place the exact code that you changed ?

**Comment by eeshadutta** [comment]
@rammanoj @srish I have removed the formatting. Please review.

File: frontend/WikiContrib-Frontend/src/App.css [hunk 1]
```diff
@@ -656,14 +656,16 @@ h3.accounts  {
 
 .ui.card.commits_load {
   width: 100% !important;
-  padding: 2% !important;
+  padding: 0.5% !important;
   box-shadow: 0 4px 16px 0 rgba(0, 0, 0, 0.25) !important;
+  margin-bottom: -0.6rem;
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/App.css:661]: You can remove this margin; with this the space between the activity boxes have reduced quite a bit.

File: frontend/WikiContrib-Frontend/src/App.css [hunk 2]
```diff
@@ -656,14 +656,16 @@ h3.accounts  {
...
 }
 
 .ui.card.commits_load:not(.loading_card):hover {
   box-shadow: none !important;
-  padding: 2% !important;
+  padding: 0.5% !important;
   border: 1px solid #C3C4C4;
+  margin-bottom: -0.6rem;
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/App.css:668]: Same as above. You can remove this margin here.

File: frontend/WikiContrib-Frontend/src/contribution.js [hunk 3]
```diff
@@ -44,7 +44,7 @@ class GenerateDay extends React.Component {
         }}
       >
         <div className="tooltip">
-          {data} {data === 1 ? 'Contribution' : 'Contributions'}
+          {data} {data === 1 ? 'Contribution ' : 'Contributions '}
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/contribution.js:47]: We don't need this change :) You can remove this.

File: frontend/WikiContrib-Frontend/src/components/activity.js [hunk 4]
```diff
@@ -136,9 +139,9 @@ class Activity extends React.Component {
                 ))}
               </React.Fragment>
             ) : (
-              <Header className="chart" style={{ textAlign: 'center' }}>
+              <Header className="chart">
                 {' '}
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/components/activity.js:143]: Not sure why you've added `{'  '}` here. We don't need this.

File: frontend/WikiContrib-Frontend/src/components/activity.js [hunk 5]
```diff
@@ -99,7 +99,10 @@ class Activity extends React.Component {
           <React.Fragment>
             {this.state.data.length !== 0 ? (
               <React.Fragment>
-                <h4 style={{marginTop:"1rem"}}>{this.props.username}'s activity on {this.props.date}</h4>
+                <Header className="chart">
+                  {' '}
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/components/activity.js:103]: Same as my comment below. Not sure why we have `{'  '}` here. We don't need this.

File: frontend/WikiContrib-Frontend/src/result.js [hunk 6]
```diff
@@ -760,13 +760,16 @@ class QueryResult extends React.Component {
...
-                  <div className="activity_wrapper">
-                    <Activity
-                      date={this.state.activity}
-                      hash={this.state.query}
-                      username={this.state.current}
-                    />
-                  </div>
+                  <Grid.Row>
+                    <Grid.Column computer={2} mobile={1} tablet={1} />
+                    <Grid.Column computer={12} mobile={14} tablet={14} style={{ paddingTop: '2rem' }}>
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/result.js:765]: Top padding of 2rem seems too much here. You can reduce it to 1rem and then also add a same bottom padding of 1rem.

**Comment by rammanoj** [comment]
@eeshadutta I am okay with the first point you mentioned in the issue. But I am not sure what does your second point imply in exact manner.
```
The boxes aren't aligned/centered appropriately. The width of the boxes can be increased.
```
Can you please elaborate it. Also can you please post the screenshot of the changes that you made ? (i.e. before and after ones).

**Pull request merged**